serverless/deploy/github/witness_pr/README.md

```diff
@@ -0,0 +1,84 @@
+# witness_pr
+
+`witness_pr` is a GitHub Action for handling incoming PRs from _witnesses_ containing a
+co-signed log checkpoint which the witness has verified is consistent with earlier
+log checkpoints it has seen.
+
+This action would be used by a serverless log which was participating in an ecosystem
```

Comment:
This presents the case as a binary thing:
 0 - witnesses not addressable; logs bundle signatures
 1 - witnesses addressable, logs don't bundle signatures

I think the log & witness issues are somewhat orthogonal, though the situation that both don't have any features is equivalent to no witnessing. TL;DR: there's a third case which may also exist: (perhaps some, maybe all) witnesses are addressable, and the log also bundles sigs as a convenience to clients.

Reply:
Yeah, this was really just to give a bit of context for how this action is currently intended to be used - I expect there'll be a bunch of refinement as we experiment with different scenarios (and maybe we can doc those in a README under `/witness` or something?)